src/pages/PassportRedemption/LyftPromo.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import styled from 'styled-components';
+import { TitleRow, Title, SubTitle, Button } from './style';
+import BikeImg from './Citibike_Lyft.png';
+import { useTranslation } from 'react-i18next';
+
+interface Props {
+    isSuccess: boolean;
+}
+
+export const LyftRewardPromo = () => {
+    const { t } = useTranslation();
+    return (
+        <LyftPromptContainer height={340}>
+            <LyftIcon src={BikeImg} />
+            <TitleRow>
+                <SubTitle bold="700">
+                    {t('lyftReward.mainBody1')}
+                    <br />
+                    <br />
+                    {t('lyftReward.mainBody2')}
+                    <br />
+                    <br />
+                    {t('lyftReward.mainBody3')}
+                </SubTitle>
+            </TitleRow>
+            <span>
+                <LyftYesButton className="button--red-filled" onClick={(e) => { }}>
+                    {t('lyftReward.yesButton')}
+                </LyftYesButton>
+                <LyftNoButton className="button--red-outlined" onClick={(e) => { }}>
+                    {t('lyftReward.noButton')}
+                </LyftNoButton>
+            </span>
+        </LyftPromptContainer>
+    );
+};
+
+export const LyftConfirmationPromo = (props: Props) => {
+    const { t } = useTranslation();
+    return (
+        <LyftPromptContainer height={285}>
+            <LyftIcon src={BikeImg} />
+            <TitleRow>
+                <SubTitle bold="700">
+                    <br />
+                    {props.isSuccess
+                        ? t('lyftReward.successBody1')
+                        : t('lyftReward.failureBody1')}
+                    <br />
+                    <br />
+                    {props.isSuccess
+                        ? t('lyftReward.successBody2')
+                        : t('lyftReward.failureBody2')}
+                </SubTitle>
+            </TitleRow>
+            <LyftCloseButton className="button--red-filled" onClick={(e) => { }}>
+                {t('lyftReward.closeButton')}
+            </LyftCloseButton>
+        </LyftPromptContainer>
+    );
+};
+
+const LyftPromptContainer = styled.div<{ height: number }>`
+  padding: 10px;
+  position: absolute;
+  width: 340px;
+  margin: 0 auto;
+  height: ${(props) => props.height}px;
+  z-index: 20;
+  top: 70px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+
```

Comment:
```suggestion
border-radius: 13px;
```

Dannnn do you hate border-radius or something. Might as well fill this empty line.

Reply:
Done